ondewo/nlu/aiservices.proto

```diff
@@ -71,9 +71,14 @@ message ExtractEntitiesRequest {
     string language_code = 3;
 }
 
-message ExtractEntitiesResponse {
-    repeated Intent.TrainingPhrase.Entity entities = 1;
+message EntityDetected{
+    Intent.TrainingPhrase.Entity entity = 1;
+    string extraction_method = 2;
+    float score = 3;
+}
 
+message ExtractEntitiesResponse {
+    repeated EntityDetected entities = 1;
```

Comment:
I would change the naming of the repeated field: You already introduced the message EntityDetected, so why not call it "entities_detected" instead of just "entities"? That way it's also clearer that it's more than just entities you're returning.